assembl/static2/js/app/utils/utilityManager.jsx

```diff
@@ -72,6 +74,21 @@ export const closeModal = () => {
   modalManager.component.setState({ showModal: false });
 };
 
+export const openShareModal = (title, routerParams, elementId, social, isFooter, footer) => {
+  const modalTitle = title;
+  const { slug, phase, themeId } = routerParams;
+  const url = `${window.location.protocol}//${window.location.host}${get('debate', {
```

Comment:
Next step here, in `routeMap`, you can add a route for thematic element, and instead use the `getFullPath` method, whilst only passing the keys needed.

Reply:
yes, try to avoid the use of `window` in components for purity/testabilty